ed25519-dalek/tests/x25519.rs

```diff
@@ -16,17 +16,8 @@ fn ed25519_to_x25519_dh() {
     let ed25519_signing_key_a = SigningKey::from_bytes(&ed25519_secret_key_a);
     let ed25519_signing_key_b = SigningKey::from_bytes(&ed25519_secret_key_b);
 
-    let scalar_a_bytes = ed25519_signing_key_a.to_scalar_bytes();
-    let scalar_b_bytes = ed25519_signing_key_b.to_scalar_bytes();
-
-    assert_eq!(
-        scalar_a_bytes,
-        hex!("357c83864f2833cb427a2ef1c00a013cfdff2768d980c0a3a520f006904de90f")
-    );
-    assert_eq!(
-        scalar_b_bytes,
-        hex!("6ebd9ed75882d52815a97585caf4790a7f6c6b3b7f821c5e259a24b02e502e11")
-    );
```

Comment:
Note: the rest of the D-H workflow is tested, however I removed checking for a particular scalar serialization here since the `Scalar` has been pre-clamped at this point.

We could check for the clamped version and make a note of it in a comment.

Reply:
Fixed.